[Merged by Bors] - [DOC] Mention FPS in log_diagnostics example (#2150)
bevyengine/bevy: examples/README.md

Example | File | Description
--- | --- | ---
`custom_diagnostic` | [`diagnostics/custom_diagnostic.rs`](./diagnostics/custom_diagnostic.rs) | Shows how to create a custom diagnostic

> `log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS / frame time diagnostics to the console

Reviewer: ```suggestion
`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS, frame time, and frame count diagnostics to the console
```

> `log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS / frame time diagnostics to the console

Reviewer: ```suggestion
`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS, frame time, and frame count diagnostics to the console
```

Author: I thought about that but discarded it because of brevity :D

> `log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs frames per second (FPS), frame time, and frame count diagnostics to the console

Reviewer 2: ```suggestion
`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs diagnostics, like frames per second (FPS), to the console
```
Diagnostics are not limited to those, we shouldn't list them as if it was exhaustive

Author: Good point indeed
